carbon-design-system/carbon
# fix(TextArea): adds character counter alert

Closes #12071 


#### Changelog

- adds `span` with `role="alert"`, visually hidden, and only announces on last 10 characters

#### Testing / Reviewing

- go to TextArea playground
    - set character counter
    - turn on VO
    - type until you reach the limit, you should see the VO announce the last 10 characters
    - if you delete the last 10 characters, VO character count should be updated
    - any number less than the last 10 should not announce anything
- all test should pass
- TextArea should still work as expected
